microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -1444,7 +1460,7 @@ The `CustomSchemeRegistration` allows registration of custom schemes in WebView2
 ##### [WinRT/C#](#tab/winrtcsharp)
 
 * [CoreWebView2CustomSchemeRegistration Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2customschemeregistration)
-* [CoreWebView2EnvironmentOptions Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions)<!-- todo: remove or comment-out this item?  no CustomSchemeRegistrations property: https://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions -->
+* [CoreWebView2EnvironmentOptions Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions)<!--todo: remove or comment-out this item?  no CustomSchemeRegistrations property: https://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions -->
```

Comment:
@peiche-jessica @novac42 - given that on WinRT, this type lacks the CustomSchemeRegistrations member, is there any reason to continue to link to this type?

https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?tabs=winrtcsharp#custom-scheme-registration

Reply:
Per discussion this one is not yet implemented in WinRT so we don't have to list the winrt version.

Reply:
removed